Integrations/integration-CyberTriage.yml

```diff
@@ -0,0 +1,256 @@
+
+        except requests.exceptions.SSLError as e:
+            ret_msg = 'Unable to verify the Cyber Triage server certificate'
+            demisto.error(traceback.format_exc())
+
+        except requests.exceptions.HTTPError as e:
+            # Format error message based on error response in json
+            ret_msg = 'No Error Message found'
+
+            if 'Error' in resp_json:
+                ret_msg = resp_json['Error']
+            elif 'message' in resp_json:
+                ret_msg = resp_json['message']
+
+            demisto.error(traceback.format_exc())
+
+        except requests.exceptions.ConnectionError as e:
+            ret_msg = 'Error while connecting to ({})'.format(rest_api)
+            demisto.error(traceback.format_exc())
+
+        except requests.exceptions.RequestException as e:
+            ret_msg = 'An unexpected error has occurred'
+            demisto.error(traceback.format_exc())
+
+        # If an exception occurred then update the exception string
+        if (e):
+            e_str = '{}: {}'.format(type(e).__name__, e)
+
+        return {
+            'response' : ret_msg,
+            'status_code' : http_status_code,
+            'exception' : e_str
+        }
+
+    # Format non 2xx responses from make_rest_call(). These are cases where an error has occurred.
+    def format_err_resp(json_resp):
+        return 'Error message: {0}\nHTTP Status code: {1}\nExceptions: {2}'.format(json_resp['response'],
+                                                                                   json_resp['status_code'],
+                                                                                   json_resp['exception'])
+
+    def test_connection():
+        response = make_rest_call(REST_APIS['check_creds'], 'get')
+
+        if IS_2XX(response['status_code']):
+            demisto.results('Test connection successful!')
+            sys.exit(0)
+        else:
+            return_error(format_err_resp(response))
+            sys.exit(1)
+
+    def triage_endpoint():
+        is_true = lambda x: x == 'yes'
+        is_hash_upload_on = is_true(CMD_ARGS['malware_hash_upload']) # arg value = 'yes' or 'no'
+        is_file_upload_on = is_true(CMD_ARGS['malware_file_upload']) # arg value = 'yes' or 'no'
+        is_fast_scan_on = not(is_true(CMD_ARGS['full_scan']))        # arg value = 'yes' or 'no'
+        endpoint = CMD_ARGS['endpoint']
+
+        # Make data dict for rest call
+        api_data = {'hostName': endpoint}
+        api_data.update({'userId': USER})
+        api_data.update({'password': PASSWORD})
+        api_data.update({'malwareScanRequested': is_hash_upload_on})
+        api_data.update({'sendContent': is_file_upload_on})
+        api_data.update({'fastScan': is_fast_scan_on})
+        api_data.update({'sendIpAddress': False})
+
+        response = make_rest_call(REST_APIS['start_collection'], 'post', api_data)
+
+        ec = {
+            'CyberTriage' : response['response']
```

Comment:
Please add the endpoint hostname/IP to the context as well under `CyberTriage`, so it can be used in playbook. In addition, please add it under the path `Endpoint.Hostname` if hostname is given or under `Endpoint.IPAddress` if IP is given.